tests/baselines/reference/vExplicitReturnType.symbols

```diff
@@ -0,0 +1,48 @@
+=== tests/cases/conformance/generators/strictGeneratorTypesExplicitReturnType.ts ===

```

Comment:
I don't see the test file or other baselines for this test. Probably left over from an earlier commit?